### PR TITLE
Module 2 JavaScript capstone project

### PR DESCRIPTION
**Pull request by oguntayomicheal**
- Data was generated from and external API
- The homepage and comment page were created
- User interactions were stored in the involvement API

### CODE REVIEW

**Comment by mirouhml** [comment]
Hi @oguntayomicheal @Harrynoqs, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive README, kindly customize it by adding the project information and your information as well.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/modules/commentFunctions.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    comDiv.innerHTML = ''
+}
+
+export const commentCounter = (array) => {
+  if(array.length){
+      return array.length
+  } else {
+     return "0"
+  }   
+}
```
Comment (mirouhml) [inline src/modules/commentFunctions.js:13]: - [ ] Please know that a counter function should look for specific DOM elements, in this case, the comments, and make the counting based on what is actually displayed on the page, and not based on the array length, kindly fix this.

File: src/modules/homeItemsCounter.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+export const homeItmes = (array) => {
+    if(array.length){
+        return array.length
+    } else {
+       return "0"
+    }
```
Comment (mirouhml) [inline src/modules/homeItemsCounter.js:7]: - [ ] Please know that a counter function should look for specific DOM elements, in this case, the home items, and make the counting based on what is actually displayed on the page, and not based on the array length, kindly fix this.

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @oguntayomicheal @Harrynoqs, 

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please know that the video you recorded should also highlight some interesting piece of code or something you built that you are very proud of, kindly record another video, presenting the same things you did in the on provided and then include the highlight I mentioned, as it is a part of the requirements, both of you should include a piece of code that you're proud of.

- [ ] Please know that the counters' functions should get covered with unit tests using [the Jest Framwework](https://jestjs.io/), kindly add that as well.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @oguntayomicheal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
### Highlights
- Amazing video presentation ✔️
- Good commit history ✔️
approval


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/modules/homeItemsCounter.js (hunk 2 above)
Comment (oguntayomicheal) [reply src/modules/homeItemsCounter.js]: Hi @mirouhml, thanks for the review. We never knew the counter function should look for specific Dom elements. All changes will be implemented

**Comment by oguntayomicheal** [comment]
Hi @uwadonat, thanks for the review.

**Pull request merged**